packages/odyssey-storybook/src/components/odyssey-mui/EmptyState/EmptyState.stories.tsx

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Meta, StoryObj } from "@storybook/react";
+import { Button, EmptyState, EmptyStateProps } from "@okta/odyssey-react-mui";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+
+const storyBookMeta: Meta<EmptyStateProps> = {
+  title: "MUI Components/EmptyState",
+  component: EmptyState,
+  argTypes: {
+    heading: {
+      control: "text",
+      description: "The main heading of the empty state",
+      type: {
+        required: true,
+        name: "string",
+      },
+    },
+
+    text: {
+      control: "text",
+      description: "A descriptive text explaining more context",
```

Comment:
```suggestion
      description: "A descriptive text explaining more context as to why we don't have data.",
```

Right? Or if not, what "more context" are we planning on people explaining with the description?

Reply:
Yes. Done.